plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/Main.java

```diff
@@ -12,12 +12,14 @@
 import java.util.List;
 import org.kohsuke.args4j.CmdLineException;
 import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.ParserProperties;
 
 public class Main {
     @SuppressFBWarnings("DM_EXIT")
     public static void main(String[] args) throws IOException {
         CliOptions options = new CliOptions();
-        CmdLineParser parser = new CmdLineParser(options);
+        ParserProperties parserProperties = ParserProperties.defaults().withUsageWidth(150);
```

Comment:
the help output was too squished by default, even on my 13" monitor.
looks a lot better like this imo